cpp/core/jni/JniWrapper.cc

```diff
@@ -179,15 +179,15 @@ class JniColumnarBatchIterator : public ColumnarBatchIterator {
  public:
   explicit JniColumnarBatchIterator(
       JNIEnv* env,
-      jobject javaserializedColumnarBatchIterator,
+      jobject javaSerializedColumnarBatchIterator,
```

Comment:
Minor: any way to simplify the name? e.g., ```jColumnarBatchItr``` better?

Reply:
fixed